lib/access-code-manager.js

```diff
@@ -22,6 +22,37 @@ module.exports = (config, promise, random, storageServiceFactory) => {
     var codeLength, secretLength, storageService;
 
     /**
+     * Verifies that an account ID is not empty.
+     *
+     * @param {string} accountId
+     * @return {Promise.<*>}
+     */
+    function checkAccountIdAsync(accountId) {
```

Comment:
Agreed.  Moved guards to the storage service.  It isn't the manager's job to do validation.  Instead that should be done at the entry point (routes, command-line tools, etc) and exit points (storage engine in this case).  0c2459b